i18n/ja/OWNERS

```diff
@@ -2,3 +2,6 @@ approvers:
   - shu-mutou
   - atoato88
 
+labels:
+- language/ja
```

Comment:
Can you add proper labels to OWNERS files in other i18n directories?

Reply:
Thank you for comment.
I will do it, after confirm that "language/ja" is attached automatically by other PR.